JavaScript1/Week2/homework.md

```diff
@@ -6,200 +6,157 @@
 - Go through the review of [last week](../Week1/review.md) (Work in progress, update this week :wrench:)
 - Go through the review of [this week](../Week2/review.md) (work in progress, update this week :nut_and_bolt:)
 
```

Comment:
Looks good and very organized by step by step building block with hints. 
Excellent work! - good to be merge.